integration-cli/docker_cli_by_digest_test.go

```diff
@@ -40,16 +35,14 @@ func setupImageWithTag(tag string) (string, error) {
 	}
 
 	// push the image
-	cmd = exec.Command(dockerBinary, "push", repoAndTag)
-	out, _, err := runCommandWithOutput(cmd)
+	out, _, err := dockerCmdWithError(c, "push", repoAndTag)
 	if err != nil {
 		return "", fmt.Errorf("pushing the image to the private registry has failed: %s, %v", out, err)
 	}
 
 	// delete our local repo that we previously tagged
-	cmd = exec.Command(dockerBinary, "rmi", repoAndTag)
-	if out, _, err := runCommandWithOutput(cmd); err != nil {
-		return "", fmt.Errorf("error deleting images prior to real test: %s, %v", out, err)
+	if rmiout, _, err := dockerCmdWithError(c, "rmi", repoAndTag); err != nil {
```

Comment:
``dockerCmdWithError`` or ``setupImageWithTag`` ? I'm note sure in what we it complicates the code that much though :blush:.